cmd/zed/zed.c

```diff
@@ -291,7 +291,7 @@ main(int argc, char *argv[])
 		rv = zed_event_service(&zcp);
 
 		/* ENODEV: When kernel module is unloaded (osx) */
-		if (rv == ENODEV)
+		if (rv != 0)
```

Comment:
Is this comment still correct?

Reply:
Sort of. ENODEV is the first error, but after that EINVAL - which is why I had to change the conditional. Some other call can "eat" the ENODEV and we spin forever. But it feels a bit stale, so rewording is probably better.